chore(rust/scanner): improve naming

hyf0 opened the pull request:
<!-- Thank you for contributing! -->

### Description

<!-- Please insert your description here and provide especially info about the "what" this PR is solving -->

hyf0 commented:
* **#1818** <a href="https://app.graphite.dev/github/pr/rolldown/rolldown/1818?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#1816** <a href="https://app.graphite.dev/github/pr/rolldown/rolldown/1816?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#1815** <a href="https://app.graphite.dev/github/pr/rolldown/rolldown/1815?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#1814** <a href="https://app.graphite.dev/github/pr/rolldown/rolldown/1814?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a>
* **#1813** <a href="https://app.graphite.dev/github/pr/rolldown/rolldown/1813?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @hyf0 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="11px" height="11px"/> <b>Graphite</b></a>
  <!-- Current dependencies on/for this PR: -->

hyf0 commented on crates/rolldown/src/ast_scanner/mod.rs:
```diff
@@ -516,13 +516,19 @@ impl<'me> AstScanner<'me> {
+        }
+      }
       None => {
         if ident.name == "module" {
```
This is wrong. User-defined `moduel` var should not be catch here.

IWANABETHATGUY commented:
Can you add a related test case or link a related issue, since your pr title starts with `fix`?

hyf0 commented:
> Can you add a related test case or link a related issue, since your pr title starts with `fix`? 

👌

hyf0 commented:
Old behavior is correct. I reverted related code.

hyf0 commented:
Change to more suitable title.

hyf0 committed:
chore(rust/scanner): improve naming (#1813)

<!-- Thank you for contributing! -->

### Description

<!-- Please insert your description here and provide especially info about the "what" this PR is solving -->
